crates/runtime/src/accelerated_table/refresh_task.rs

```diff
@@ -153,7 +155,7 @@ impl RefreshTask {
         let (start_time, data_update) = match get_data_update_result {
             Ok((start_time, data_update)) => (start_time, data_update),
             Err(e) => {
-                tracing::warn!("Error getting update for dataset {dataset_name}: {e}");
+                tracing::error!("Failed to load data for dataset{dataset_name}: {e}");
```

Comment:
This is changed intentionally to fix duplicate error tracing bug: additional warning message after error message.